### PR TITLE
Poezio

### PR DESCRIPTION
**Pull request by delehef**
Adds [poezio](http://poez.io/en/) and its dependencies.

### CODE REVIEW

File: srcpkgs/python3-aiodns/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python-aiodns'
```
Comment (chneukirchen) [inline srcpkgs/python3-aiodns/template]: Wrong name.

File: srcpkgs/python3-slixmpp/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-slixmpp'
+pkgname=python3-slixmpp
+version=1.2.1
+revision=1
+noarch=yes
+wrksrc="slixmpp-${version}"
+short_desc="Slixmpp is an elegant Python library for XMPP"
```
Comment (chneukirchen) [inline srcpkgs/python3-slixmpp/template]: Don't use "XXX is..."

**Comment by chneukirchen** [comment]
Cross-build needs fixing, else LGTM.

**Comment by chneukirchen** [comment]
The culprit is `-I/usr/include/python3.5m`.

**Comment by delehef** [comment]
Thanks, I fixed the two typos.

However, I don't have an ARM machine to experiment further with the ARM build and I'm only using the standard `python3-module`, so I don't see how I could fix it.

Should I limit architectures to i686 & x86_64?

**Comment by chneukirchen** [comment]
Fixed.

**Pull request closed without merging**